### ISSUE TITLE
Does glTF 1.0 allow application-specific semantics?

### ISSUE THREAD

**Issue by donmccurdy** [KhronosGroup/glTF#1264]
I'm https://github.com/mrdoob/three.js/pull/13339, and struggling to understand how/if custom semantics may be used in glTF 1.0. Are they allowed? Language below suggests values not listed are invalid, to me:

> Identifies a parameter with a well-known meaning.

>  Attribute semantics include "POSITION", "NORMAL", "TEXCOORD", "COLOR", "JOINT", and "WEIGHT". Attribute semantics can be of the form [semantic]_[set_index], e.g., "TEXCOORD_0".

The draft glTF 2.0 `KHR_technique_webgl` spec does include language about application-specific semantics.

**Comment by lexaknyazev** [comment]
glTF **1.0** doesn't support them. 
That functionality has been initially proposed as a part of glTF **1.0.1** and later glTF **1.1** which eventually became glTF **2.0**.

**Comment by donmccurdy** [comment]
Ok, thanks! That clarifies things then.

**Comment by 1d2d3d** [comment]
I couldn't find the spot in the spec where custom attributes are disallowed. I did find this: 

Attribute semantics include "POSITION", "NORMAL", "TEXCOORD", "COLOR", "JOINT", and "WEIGHT". Attribute semantics can be of the form [semantic]_[set_index], e.g., "TEXCOORD_0".

I believe that means there are no custom attribute _semantics_ (I saw "well-known value" attached to attribute semantics in the spec.) But that would mean any attribute whose semantic follows the [semantic]_[set_index] pattern (e.g., an attribute named "myColor" with semantic "COLOR_2") would be valid.

Of course I could be missing something.
